### PR TITLE
Fix member check on autoload

### PR DESCRIPTION
**Pull request by albertodonato**
Fix erroneous check for 'hl-nics in erc-modules.

### CODE REVIEW

File: erc-hl-nicks.el [hunk 1]
```diff
@@ -354,7 +354,7 @@
 ;; For first time use
 ;;;###autoload
 (when (and (boundp 'erc-modules)
-           (not (member 'hl-nicks 'erc-modules)))
+           (not (member 'hl-nicks erc-modules)))
```
Comment (sparkiegeek) [inline erc-hl-nicks.el]: Why bother with this check? `add-to-list` already checks for presence in the list before adding it

Comment (albertodonato) [inline erc-hl-nicks.el]: Good point, fixed.

**Comment by leathekd** [comment]
Thanks for contributing!

**Comment by albertodonato** [comment]
Thanks for merging! Would you mind also bumping the revision and pushing to the repos?

**Comment by leathekd** [comment]
Sure thing.  I just updated Marmalade and melpa/melpa-stable should pick up the changes soon.

**Pull request merged**